cms/envs/aws_migrate.py

```diff
@@ -1,30 +0,0 @@
-"""
```

Comment:
Thanks, I was working on a branch in configuration that I had made just before that landed. Rebasing will probably fix it.